src/main/java/org/wso2/carbon/connector/util/ResultPayloadCreate.java

```diff
@@ -167,7 +167,9 @@ public static boolean buildFile(FileObject file, MessageContext msgCtx, String c
                         contentType, axis2MsgCtx);
             }
             //We need this to build the complete message before closing the stream
-            //noinspection ResultOfMethodCallIgnored
+            if ("false".equals(streaming) || StringUtils.isEmpty(streaming)) {
```

Comment:
Shall we swap the two statements since the statement is read from left to right. It would make sense to first check whether the string is empty

Reply:
Fixed in https://github.com/wso2-extensions/esb-connector-file/pull/28